src/lightning_app/core/app.py

```diff
@@ -345,6 +397,8 @@ def run_once(self):
         elif self.stage == AppStage.RESTARTING:
             return self._apply_restarting()
 
+        self.apply_commands()
```

Comment:
This involves uploading a command, I would not do that as part of the loop. Can’t we do that outside the loop?

Reply:
Maybe the commands can be considered as a property of the app/flow and cannot be updated one time the app has started so the commands will be collected only once during app start?

Reply:
Yes, moved to this approach for now. This would prevent dynamic commands, but it is fine for now.